AB#21 - create Github action to build solution upon new commits

bytetum opened the pull request.

bytetum committed:
Create dotnet.yml

popemkt commented on .github/workflows/dotnet.yml:
```diff
@@ -0,0 +1,28 @@
+
+on:
+  push:
+    branches: [ "main" ]
```
Need Dev branch as well as in ticket description

thanhhxse replied:
> Need Dev branch as well as in ticket description

fixed

bytetum committed:
Update dotnet.yml

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions

using yaml branches filter to trigger github action on either main or dev branch
